src/frontend/api/oppsummeringvedtak.ts

```diff
@@ -1,27 +1,18 @@
 import { IFagsak } from '../typer/fagsak';
-import { Ressurs, RessursStatus } from '../typer/ressurs';
+import { Ressurs } from '../typer/ressurs';
 import { ISaksbehandler } from '../typer/saksbehandler';
 import { axiosRequest } from './axios';
+import { aktivVedtak } from './fagsak';
 
 export const hentAktivVedtaksbrev = (
     fagsak: IFagsak,
     innloggetSaksbehandler?: ISaksbehandler
 ): Promise<Ressurs<string>> => {
-    const aktivBehandling = fagsak.behandlinger.find(b => b.aktiv);
-
-    if (aktivBehandling === undefined) {
-        return new Promise(resolve =>
-            resolve({
-                melding: 'Internal Error: No active behandling',
-                status: RessursStatus.FEILET,
-            })
-        );
-    }
-
+    const vedtak = aktivVedtak(fagsak);
```

Comment:
Can't this return null? We should probably handle it before we use it to call ba-sak.

Reply:
This shouldn't be null here, otherwise it is a bug.